domain/src/main/kotlin/no/nav/su/se/bakover/domain/revurdering/Revurdering.kt

```diff
@@ -650,6 +670,7 @@ sealed class RevurderingTilAttestering : Revurdering() {
 
         fun tilIverksatt(
             attestant: NavIdentBruker.Attestant,
+            clock: Clock = Clock.systemUTC(),
```

Comment:
Kunne evt tatt inn en `Attestering.Iverksatt`? Tilsvarende som gjøres for underkjent.